backend/src/middleware/errorHandler.ts

```
@@ -0,0 +1,280 @@
+export const notFoundHandler = (req: Request, res: Response): void => {
+  const apiError: ApiError = {
+    code: 'NOT_FOUND',
+    message: `Route ${req.originalUrl} not found`,
+    statusCode: 404,
+  };
+
+  sendErrorResponse(res, apiError);
+};
+
+// Async error wrapper
+export const asyncHandler = (fn: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)